stats/gRPC.md

```diff
@@ -0,0 +1,125 @@
+| grpc.canonical_status | gRPC canonical status code.                                   |
+
+### Default views
+
+The following set of views are considered minimum required to monitor client side performance:
+
+| View name                                   | Measure suffix              | Aggregation  | Tags suffix                     |
+|---------------------------------------------|-----------------------------|--------------|---------------------------------|
+| grpc.io/client/started_count                | started_count               | count        | client_method                   |
+| grpc.io/client/request_bytes                | request_bytes               | distribution | client_method                   |
+| grpc.io/client/response_bytes               | response_bytes              | distribution | client_method                   |
+| grpc.io/client/latency                      | latency                     | distribution | client_method                   |
+| grpc.io/client/finished_count               | latency                     | count        | client_method, canonical_status |
+
+### Extra views
+
+The following set of views are considered useful but not mandatory to monitor client side performance:
+
+| View name                                   | Measure suffix              | Aggregation  | Tags suffix                     |
+|---------------------------------------------|-----------------------------|--------------|---------------------------------|
+| grpc.io/client/request_count                | request_count               | distribution | client_method                   |
+| grpc.io/client/uncompressed_request_bytes   | uncompressed_request_bytes  | distribution | client_method                   |
+| grpc.io/client/response_count               | response_count              | distribution | client_method                   |
+| grpc.io/client/uncompressed_response_bytes  | uncompressed_response_bytes | distribution | client_method                   |
+| grpc.io/client/server_latency               | server_latency              | distribution | client_method                   |
+
+## Server
+
+Server stats are recorded for each individual gRPC request. All measurements except `grpc.io/server/started_count` must be recorded at the end of each gRPC request.
+
+| Measure name                               | Unit | Description                                                                                   |
+|--------------------------------------------|------|-----------------------------------------------------------------------------------------------|
+| grpc.io/server/started_count               |      | Number of all server requests started. Records "1" at the beginning of each gRPC request.     |
+| grpc.io/server/request_count               |      | Number of request messages in a gRPC request.                                                 |
```

Comment:
clarify that this will be always "1" for non-streaming requests.

Reply:
done.